satpy/readers/__init__.py

```diff
@@ -315,10 +312,13 @@ def available_readers(as_dict=False):
         try:
             reader_info = read_reader_config(reader_configs)
         except (KeyError, IOError, yaml.YAMLError):
-            LOG.warning("Could not import reader config from: %s", reader_configs)
```

Comment:
Could you explain why you removed this?

Reply:
The person who made the original request recommended that we "suppress [the output] when the reader config is not found as a first time user this creates a lot of clutter."

Reply:
Good thinking! Could you just make it a `debug` instead? That way, regular users won't see it but devs will still get the info.